.rubocop.yml

```diff
@@ -33,6 +33,11 @@ Metrics/MethodLength:
 Metrics/ModuleLength:
   Max: 120
 
+# Disabled since it only applies to Rails 5. Reenable when we migrate our apps to Rails 5.
```

Comment:
Ok, never-mind, `redirect_back` was supposed to go in the controller, not the controller spec. This is working now.